packages/core/src/services/chains/ProviderProcessor/saveOrPublishProviderLogs.ts

```diff
@@ -13,7 +11,7 @@ export type LogData<T extends StreamType> = T extends 'text'
   ? Awaited<TextProviderLogsData>
   : T extends 'object'
     ? Awaited<ObjectProviderLogsData>
-    : never
+    : unknown
```

Comment:
hmm why? should never happen no?

Reply:
`tsc` failed with `never` https://chatgpt.com/share/e/66ffb37b-d09c-800a-8d8c-45618c0b8ddd